HLTrigger/Muon/src/HLTL1MuonNoL2Selector.cc

```diff
@@ -53,7 +53,6 @@ HLTL1MuonNoL2Selector::fillDescriptions(edm::ConfigurationDescriptions& descript
   edm::ParameterSetDescription desc;
   desc.add<edm::InputTag>("InputObjects",edm::InputTag(""));
   desc.add<edm::InputTag>("L2CandTag",edm::InputTag("hltL2MuonCandidates"));
-  desc.add<edm::InputTag>("L1CandTag",edm::InputTag(""));
```

Comment:
We've removed this because it is not used anywhere in the code, but the removal of this line require parsing the code to ConfDB as the ConfDB version is still asking for this parameter.  How do we want to proceed?